src/sentry/celery.py

```diff
@@ -83,6 +84,10 @@ def good_use_of_pickle_or_bad_use_of_pickle(task, args, kwargs):
 class SentryTask(Task):
     Request = "sentry.celery:SentryRequest"
 
+    def delay(self, *args, **kwargs):
+        kwargs["__start_time"] = datetime.now().timestamp() * 1000
```

Comment:
Nit: Should we just do the `* 1000` when calculating the duration so it's not in two places?